xc/common/cmake/vivado.cmake

```diff
@@ -273,15 +280,16 @@ function(ADD_VIVADO_TARGET)
       WORK_DIR ${WORK_DIR}
       DEPS ${DEPS}
       BITSTREAM ${BITSTREAM}
-      MAKE_DIFF_FASM)
```

Comment:
Again, you are not controlling whether we check the Verilog/XDC via a CMake cache variable, but instead requiring changes to the `.cmake`.  We should be able to control this with a CMake cache variable.